docs/machine-learning/tutorials/image-classification.md

```diff
@@ -111,336 +111,249 @@ toaster2.png	appliance
+    [!code-csharp[AddTrainer](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#AddTrainer)]
 
-Add the following code to display your data and predictions as the next lines in the `ReuseAndTuneInceptionModel()` method:
+1. Add the estimator to map the predicted key value back into a string:
 
-[!code-csharp[CallDisplayResults1](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#CallDisplayResults1)]
+    [!code-csharp[MapKeyToValue](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#MapKeyToValue)]
 
-Once you have the prediction set, the [Evaluate()](xref:Microsoft.ML.RecommendationCatalog.Evaluate%2A) method:
+## Train the model
 
-* Assesses the model (compares the predicted values with the actual dataset `Labels`).
+1. Load the training data using the `MLContext.Data.LoadFromTextFile` wrapper. Add the following code as the next line in the `GenerateModel()` method:
 
-* Returns the model performance metrics.
+    [!code-csharp[LoadData](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#LoadData "Load the data")]
 
-Add the following code to the `ReuseAndTuneInceptionModel()` method as the next line:
+    Data in ML.NET is represented as an [IDataView class](xref:Microsoft.ML.IDataView). `IDataView` is a flexible, efficient way of describing tabular data (numeric and text). Data can be loaded from a text file or in real time (for example, SQL database or log files) to an `IDataView` object.
 
-[!code-csharp[Evaluate](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#Evaluate)]
+1. Train the model with the data loaded above:
 
-The following metrics are evaluated for image classification:
+    [!code-csharp[TrainModel](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#TrainModel)]
 
-* `Log-loss` - see [Log Loss](../resources/glossary.md#log-loss). You want Log-loss to be as close to zero as possible.
+    The `Fit()` method trains your model by applying the training dataset to the pipeline.
 
-* `Per class Log-loss`. You want per class Log-loss to be as close to zero as possible.
+## Evaluate the accuracy of the model
 
-Use the following code to display the metrics, share the results, and then act on them:
+1. Load and transform the test data, by adding the following code to the next line of the `GenerateModel` method:
 
-[!code-csharp[DisplayMetrics](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#DisplayMetrics)]
+    [!code-csharp[LoadAndTransformTestData](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#LoadAndTransformTestData "Load and transform test data")]
 
- Add the following code to return the trained model as the next line:
+    There are a few sample images that you can use to evaluate the model. Like the training data, these need to be loaded into an IDataView, so that they can be transformed by the model.
```

Comment:
```suggestion
    There are a few sample images that you can use to evaluate the model. Like the training data, these need to be loaded into an `IDataView`, so that they can be transformed by the model.
```

Reply:
Fixed.